public/pages/CreateMonitor/components/MonitorDefinitionCards/MonitorDefinitionCard.js

```diff
@@ -26,10 +26,11 @@
 
 import React from 'react';
 import _ from 'lodash';
-import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
```

Comment:
Should we look out for alternative to elastic/eui? It's been used extensively across Open-Search project. Even though it has a SSPL license but it's still an elastic product. WDYT?

Reply:
This is an interesting question, I should find some time to discuss with our UX designer to learn more about the topic and what we would benefit/sacrifice from this.

Currently I'm thinking of the potential approach to move away from EUI would be:
a) Entire update starting from core OpenSearch Dashboard
- Would be a huge amount of work and time to do testings. 
- Probably need more research to see what library shows the visuals better and supports the forms/workflows on pages well.

b) Start the change for newer pages developed
- Inconsistent look comparing to older pages and core OpenSearch Dashboards.
- Could be used if the function of component does not exist in other pages within the plugin. (Such as the create monitor page in alerting uses React visual to show preview graph)      

Anyways, at least for now we want to use components and interface that users are familiar with and stay consistent with other pages.😄

Reply:
I agree on the part that this has to come up as a new project and as a part of larger migration effort. But maybe something worth investing.